### DISCUSSION TITLE
I compiled SeisSol with ASAGI ON, why I encourter this problem

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#567] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/104721358/171611975-f28dea4c-5194-478c-a9af-4de8d9ab78c2.png)

**Comment by sebwolf-de** [comment, marked as answer]
Did you `./generate_ASAGI_file.sh`?

**Reply by smallriver666** [reply]
> Did you `./generate_ASAGI_file.sh`?
But it is not an executable program.
![image](https://user-images.githubusercontent.com/104721358/176363227-32d7662d-f56a-4ad9-9454-0460a81c1d81.png)

**Reply by krenzland** [reply]
See https://github.com/SeisSol/Examples/pull/45
For now you can run it with `bash generate_ASAGI_file.sh`

**Reply by smallriver666** [reply]
> See [SeisSol/Examples#45](https://github.com/SeisSol/Examples/pull/45) For now you can run it with `bash generate_ASAGI_file.sh`

Thank you, I will have a try.
